rmf_fleet_adapter/src/rmf_fleet_adapter/phases/RequestLift.cpp

```diff
@@ -235,6 +235,19 @@ void RequestLift::ActivePhase::_init_obs()
             agv::Destination::Implementation::get(*me->_data.localize_after)
             .position = me->_context->position();
 
+            rmf_traffic::agv::Graph::LiftPropertiesPtr in_lift;
+            const auto graph = me->_context->navigation_graph();
+            for (const auto& lift : graph.all_known_lifts())
```

Comment:
You should be able to use `graph.find_known_lift(lift->name())` instead of using this for-loop. Let me know if you run into any issue with that.

Reply:
Ah thanks for pointing me to this API! It works great